### PR TITLE
new ui for email verification due to change in the email address

### PR DESCRIPTION
**Pull request by salmanmanekia**
#### Summary
This PR updates the email UI which is being sent when the user updates his email address and needs to verify his email.
  

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-44790

#### Screenshots

|  before  |  after  |
|----|----|
|  Screenshot 2022-09-19 at 21 30 04 | Screenshot 2022-10-13 at 16 51 30 |

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: templates/email_change_verify_body.html [hunk 1]
```diff
@@ -1,46 +1,203 @@
...
+                                    <div class="mj-column-per-100 mj-outlook-group-fix" style="font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;">
+                                        <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="vertical-align:top;" width="100%">
+                                            <tbody>
+                                            <tr>
+                                                <td align="center" class="info" style="font-size:0px;padding:0px;word-break:break-word;">
+                                                    <p style="font-family: 'Open Sans';font-style: normal;font-weight: 600;font-size: 16px; line-height: 24px; margin-top: 56px; margin-bottom: 2px; color: #3F4350;">{{.Props.QuestionTitle}}</p>
+                                                    <p style="font-family: 'Open Sans';font-style: normal;font-weight: 400;font-size: 14px;line-height: 20px;margin-top: 2px; color: #3F4350;">{{.Props.EmailInfo1}}<a href='mailto:{{.Props.SupportEmail}}' style='text-decoration: none; color:#2389D7;'>{{.Props.SupportEmail}}</a></p>
```
Comment (julmondragon) [inline templates/email_change_verify_body.html:161]: It looks like these two `p` elements are not taking the expected font family:

Screen Shot 2022-09-28 at 4 50 39 PM

Maybe just need to be set as in other elements of the template: `font-family: Open Sans, sans-serif;`

**Review by julmondragon — CHANGES_REQUESTED** [review]
Looks good! Just one item requiring attention.

**Review by AGMETEOR — APPROVED** [review]
LGTM pending Julian's comments

**Review by stevemudie — APPROVED** [review]
Looks good @salmanmanekia!

**Review by julmondragon — APPROVED** [review]
Thanks @salmanmanekia !

**Comment by annelieseklein** [comment]
Looks great! @salmanmanekia each individual font type needs to be updated to match the design. 
1. heading = Metropolis, semi-bold, 28px
2. Body = open sans, 16 px (16px away from the heading as well)

Screen Shot 2022-10-03 at 9 27 33 AM

**Comment by salmanmanekia** [comment]
> Looks great! @salmanmanekia each individual font type needs to be updated to match the design.
> 
> 1. heading = Metropolis, semi-bold, 28px
> 2. Body = open sans, 16 px (16px away from the heading as well)
> 
> Screen Shot 2022-10-03 at 9 27 33 AM

Hello @annelieseklein, The screen that you have attached is for email verification when a new user is created while the changes in this PR is related to email verification when an email change is requested.

**Comment by annelieseklein** [comment]
@salmanmanekia for sure, sorry I was referring to the font types which should be the same

**Review by neallred — APPROVED** [review]
LGTM, thanks @salmanmanekia !

**Review by annelieseklein — APPROVED** [review]

**Comment by annelieseklein** [comment]
@salmanmanekia can you share the updated image of the email please?

**Comment by annelieseklein** [comment]
@salmanmanekia Great, thanks! "Questions?" should be in semibold

**Pull request merged**